src/client/msgBusClient.py

```diff
@@ -88,9 +59,39 @@ def hit_kibana(message, count):
 
         es.index(index="alerts", doc_type="_doc", id=count, body=document)
 
-        #print(es.get(index="alerts", doc_type="_doc", id=count))
-
-
-if __name__ == '__main__':
-    zs = ZClient()
-    zs.receive_message()
+# Argument parsing
+ap = argparse.ArgumentParser()
+ap.add_argument('config', help='JSON configuration')
+ap.add_argument('-t', '--topic', default='BLAS', help='Topic')
+ap.add_argument('-p', '--print', default=False, action='store_true',
+                help='Print the received message')
+args = ap.parse_args()
+
+with open(args.config, 'r') as f:
+    config = json.load(f)
+
+try:
+    print('[INFO] Initializing message bus context')
+    msgbus = mb.MsgbusContext(config)
+
+    print(f'[INFO] Initializing subscriber for topic \'{args.topic}\'')
+    subscriber = msgbus.new_subscriber(args.topic)
+
+    print('[INFO] Running...')
+    while True:
+        msg = subscriber.recv()
+        meta_data, blob = msg
+        if meta_data is not None:
+            if args.print:
+                payload = msg.get_name()+","+meta_data["message"]
+                print(f'[INFO] RECEIVED: message: {payload}')
+                logging.info('{}   - {}'.format(payload, time.strftime("%Y-%m-%d %H:%M")))
+                count +=1
+                if len(payload.split(",")) == 7:
+                    Esearch().hit_kibana(payload, count)
```

Comment:
Maybe a constant like KIBANA_PAYLOAD_SIZE could be defined to avoid a hard-coded value.